test/jasmine/tests/shapes_test.js

```diff
@@ -75,33 +159,32 @@ describe('Test shapes:', function() {
 
     describe('Plotly.relayout', function() {
         it('should be able to add a shape', function(done) {
-            var pathCount = countShapePaths();
+            var pathCount = countShapePathsInUpperLayer();
             var index = countShapes(gd);
             var shape = getRandomShape();
 
-            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(function() {
-                expect(countShapeLayers()).toEqual(1);
-                expect(countShapePaths()).toEqual(pathCount + 1);
+            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(function()
+            {
```

Comment:
we should really pick an `eslint` [brace-style](http://eslint.org/docs/rules/brace-style).

I'd vote for `Stroustrup`, I know @mdtusz would vote for `1tbs`, but no one on our would vote for `Allman`.

So please, bring up that `{` on that same line as `function()`.